cmd/cstorvolumeclaim/cstorvolumeclaim.go

```diff
@@ -437,3 +442,27 @@ func randomizePoolList(list *apis.CStorPoolInstanceList) *apis.CStorPoolInstance
 
 	return res
 }
+
+// CVCWithVersionDetails build cvc resource with required version
+// details and volume references
+func CVCWithVersionDetails(
```

Comment:
this function is doing more than what the name of function suggest

Reply:
what about  `WithVersionAndReferenceDetails` ?